config/initializers/marshal_autoloader.rb

```diff
@@ -3,7 +3,7 @@
 # processes or load just the classes that are marshaled, which may be far less classes and locations than when
 # we originally wrote this initializer.
 module MarshalAutoloader
-  def load(data)
+  def load(data, _proc = nil)
```

Comment:
haha, I was just about to comment to add that = nil!

Great minds think alike.